lametro/templatetags/lametro_extras.py

```diff
@@ -175,5 +176,28 @@ def get_highlighted_attachment_text(context, ocd_id):
     attachment_text = ' '.join(d.full_text for d in bill.documents.all() if d.full_text)
 
     highlight = ExactHighlighter(context['query'])
-    
+
     return highlight.highlight(attachment_text)
+
+
+def get_event_occured(action):
+    try:
+        event = LAMetroEvent.objects.get(participants__entity_type='organization',\
+                                        participants__entity_name=action.organization, \
+                                        start_time__date=action.date.date())
+    except MultipleObjectsReturned:
+        bill = action.bill()
+        event = LAMetroEvent.objects.get(participants__entity_type='organization',\
+                                        participants__entity_name=action.organization, \
+                                        start_time__date=action.date.date()) \
+                                        .get(agenda_items__bill=bill)
```

Comment:
this seems like a smart filter to apply all the time. is there a reason to only check if there are multiple objects returned?

Reply:
clarification: why do you only check to see whether the bill is on an event agenda if multiple events are returned? it seems like a good thing to filter on regardless.

Reply:
right now in the first block, trying to use `action.bill()` doesn't work because it tries to call `LAMetroBill` and that throws an error. i could condense it this way:
```
bill = action.bill()
event = LAMetroEvent.objects.filter(participants__entity_type='organization',\
                                        participants__entity_name=action.organization, \
                                        start_time__date=action.date.date()) \
                                        .get(agenda_items__bill=action._bill)
```

i think i misunderstood the privacy of `._bill` and avoided using it.

Reply:
what is the difference between `LAMetroBill` and `_bill`? what error does `action.bill()` throw?

Reply:
here's the error:
```
----> 1 action.bill()
TypeError: 'LAMetroBill' object is not callable
```

[`action._bill`](https://github.com/datamade/django-councilmatic/blob/master/councilmatic_core/models.py#L522) is a field that links to the underlying `Bill` object. `action.bill()` calls the subclass ([`override_relation()`](https://github.com/datamade/django-councilmatic/blob/master/councilmatic_core/models.py#L34)), if any, instead—`LAMetroBill` in this case.

**ALL THAT BEING SAID**..............the problem was me _calling_ `action.bill()` instead of using it properly, like `action.bill`.